scheduling-manager/src/test/java/com/amazonaws/blox/scheduling/manager/ManagerHandlerTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.scheduling.manager;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.amazonaws.blox.dataservicemodel.v1.client.DataService;
+import com.amazonaws.blox.dataservicemodel.v1.model.operations.ListEnvironmentsRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.operations.ListEnvironmentsResponse;
+import com.amazonaws.blox.lambda.LambdaFunction;
+import com.amazonaws.blox.scheduling.scheduler.SchedulerInput;
+import com.amazonaws.blox.scheduling.scheduler.SchedulerOutput;
+import com.amazonaws.blox.scheduling.state.ECSState;
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ManagerHandlerTest {
+
+  ArgumentCaptor<SchedulerInput> schedulerArgument = ArgumentCaptor.forClass(SchedulerInput.class);
+  @Mock private LambdaFunction<SchedulerInput, SchedulerOutput> scheduler;
+  @Mock private ECSState ecs;
+  @Mock private DataService dataService;
+
+  @Test
+  @SuppressWarnings("unchecked")
+  public void invokesSchedulerForAllEnvironments() {
+    when(dataService.listEnvironments(new ListEnvironmentsRequest("arn:::::cluster1")))
```

Comment:
let's move the strings into constants

Reply:
Ah, did that for some other cases, didn't get around to this one.